wolfcrypt/src/tfm.c

```diff
@@ -1840,8 +1840,8 @@ int fp_exptmod_nb(exptModNb_t* nb, fp_int* G, fp_int* X, fp_int* P, fp_int* Y)
   #ifdef WC_NO_CACHE_RESISTANT
     err = fp_sqr(&nb->R[nb->y], &nb->R[nb->y]);
   #else
-    fp_copy((fp_int*) ( ((wolfssl_word)&nb->R[0] & wc_off_on_addr[nb->y^1]) +
-                        ((wolfssl_word)&nb->R[1] & wc_off_on_addr[nb->y]) ),
+    fp_copy((fp_int*) ( ((wc_ptr_t)&nb->R[0] & wc_off_on_addr[nb->y^1]) +
+                        ((wc_ptr_t)&nb->R[1] & wc_off_on_addr[nb->y]) ),
```

Comment:
Is there something I'm missing or could we simplify to this and remove `wc_off_on_addr`?

```suggestion
    fp_copy((fp_int*) ( &nb->R[nb->y]),
```

Reply:
I replied... it didn't save. This is here for timing resistance and needs to remain the same.

Reply:
interesting, I didn't include the change as I thought there was something. Any resource on why one is why one is timing resistant and the other isn't?